ormar/models/metaclass.py

```diff
@@ -222,10 +222,6 @@ def update_attrs_from_base_meta(  # noqa: CCR001
                     model_fields=model_fields,
                     parent_value=parent_value,
                 )
-                parent_value = [
-                    ormar.fields.constraints.UniqueColumns(*x._pending_colargs)
-                    for x in parent_value
-                ]
```

Comment:
You cannot remove this piece, this is why some of the postgress fails.

Basically, you have to copy the constraint unpacking it's values (here and for index it is `_pending_colargs`, not sure for CheckColumns. If you take the parent value as is (as it is now) the constraint is already bound to the previously created already existing table and will fail.